From c7632351cfafcf751e9e8a9c962428279139cfca Mon Sep 17 00:00:00 2001
From: Robert Chiras <robert.chiras@nxp.com>
Date: Tue, 9 Jan 2018 11:18:34 +0200
Subject: [PATCH 3194/5242] MLK-17275-15: arm64: dts: fsl-imx8qxp-lpddr4-arm2:
 Remove dsi/lvds specific dts files

commit  32e8ecb5f021eead8661c09b2faf731888a66997 from
https://source.codeaurora.org/external/imx/linux-imx.git

Delete the dst files specific to dsi/lvds nodes for the 8QXP LPDDR4
platform.

Signed-off-by: Robert Chiras <robert.chiras@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/Makefile             |    2 -
 .../fsl-imx8qxp-lpddr4-arm2-dsi-adv7535.dts        |   95 ------------------
 .../freescale/fsl-imx8qxp-lpddr4-arm2-it6263.dts   |  103 --------------------
 3 files changed, 200 deletions(-)
 delete mode 100644 arch/arm64/boot/dts/freescale/fsl-imx8qxp-lpddr4-arm2-dsi-adv7535.dts
 delete mode 100644 arch/arm64/boot/dts/freescale/fsl-imx8qxp-lpddr4-arm2-it6263.dts

diff --git a/arch/arm64/boot/dts/freescale/Makefile b/arch/arm64/boot/dts/freescale/Makefile
index 4611ae4..de7394e 100644
--- a/arch/arm64/boot/dts/freescale/Makefile
+++ b/arch/arm64/boot/dts/freescale/Makefile
@@ -41,13 +41,11 @@ dtb-$(CONFIG_ARCH_FSL_IMX8QXP) += fsl-imx8qxp-lpddr4-arm2.dtb \
 				  fsl-imx8qxp-mek-dsi-rm67191.dtb \
 				  fsl-imx8qxp-lpddr4-arm2-enet2.dtb \
 				  fsl-imx8qxp-lpddr4-arm2-gpmi-nand.dtb \
-				  fsl-imx8qxp-lpddr4-arm2-it6263.dtb \
 				  fsl-imx8qxp-lpddr4-arm2-lpspi.dtb \
 				  fsl-imx8qxp-lpddr4-arm2-spdif.dtb \
 				  fsl-imx8qxp-lpddr4-arm2-mlb.dtb \
 				  fsl-imx8qxp-lpddr4-arm2-mqs.dtb \
 				  fsl-imx8qxp-lpddr4-arm2-wm8962.dtb \
-				  fsl-imx8qxp-lpddr4-arm2-dsi-adv7535.dtb \
 				  fsl-imx8qxp-lpddr4-arm2-dsi-rm67191.dtb
 dtb-$(CONFIG_ARCH_FSL_IMX8MQ) += fsl-imx8mq-ddr3l-arm2.dtb \
 				 fsl-imx8mq-ddr4-arm2.dtb \
diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-lpddr4-arm2-dsi-adv7535.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-lpddr4-arm2-dsi-adv7535.dts
deleted file mode 100644
index d7cd1b7..0000000
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-lpddr4-arm2-dsi-adv7535.dts
+++ /dev/null
@@ -1,95 +0,0 @@
-/*
- * Copyright 2017 NXP
- *
- * This program is free software; you can redistribute it and/or
- * modify it under the terms of the GNU General Public License
- * as published by the Free Software Foundation; either version 2
- * of the License, or (at your option) any later version.
- *
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU General Public License for more details.
- */
-
-#include "fsl-imx8qxp-lpddr4-arm2.dts"
-
-&i2c0_mipi_lvds0 {
-	#address-cells = <1>;
-	#size-cells = <0>;
-	pinctrl-names = "default";
-	pinctrl-0 = <&pinctrl_i2c0_mipi_lvds0>;
-	clock-frequency = <100000>;
-	status = "okay";
-
-	adv_bridge1: adv7535@3d {
-		compatible = "adi,adv7535", "adi,adv7533";
-		reg = <0x3d>;
-		adi,dsi-lanes = <4>;
-		status = "okay";
-
-		port {
-			adv7535_1_in: endpoint {
-				remote-endpoint = <&mipi_dsi_bridge1_out>;
-			};
-		};
-	};
-};
-
-&mipi_dsi_phy1 {
-	status = "okay";
-};
-
-&mipi_dsi1 {
-	status = "okay";
-};
-
-&mipi_dsi_bridge1 {
-	status = "okay";
-
-	port@1 {
-		mipi_dsi_bridge1_out: endpoint {
-			remote-endpoint = <&adv7535_1_in>;
-		};
-	};
-};
-
-&i2c0_mipi_lvds1 {
-	#address-cells = <1>;
-	#size-cells = <0>;
-	pinctrl-names = "default";
-	pinctrl-0 = <&pinctrl_i2c0_mipi_lvds1>;
-	clock-frequency = <100000>;
-	status = "okay";
-
-	adv_bridge2: adv7535@3d {
-		compatible = "adi,adv7535", "adi,adv7533";
-		reg = <0x3d>;
-		adi,dsi-lanes = <4>;
-		status = "okay";
-
-		port {
-			adv7535_2_in: endpoint {
-				remote-endpoint = <&mipi_dsi_bridge2_out>;
-			};
-		};
-	};
-};
-
-&mipi_dsi_phy2 {
-	status = "okay";
-};
-
-&mipi_dsi2 {
-	status = "okay";
-};
-
-&mipi_dsi_bridge2 {
-	status = "okay";
-
-	port@1 {
-		mipi_dsi_bridge2_out: endpoint {
-			remote-endpoint = <&adv7535_2_in>;
-		};
-	};
-};
diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-lpddr4-arm2-it6263.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-lpddr4-arm2-it6263.dts
deleted file mode 100644
index 7299806..0000000
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-lpddr4-arm2-it6263.dts
+++ /dev/null
@@ -1,103 +0,0 @@
-/*
- * Copyright 2017 NXP
- *
- * This program is free software; you can redistribute it and/or
- * modify it under the terms of the GNU General Public License
- * as published by the Free Software Foundation; either version 2
- * of the License, or (at your option) any later version.
- *
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU General Public License for more details.
- */
-
-#include "fsl-imx8qxp-lpddr4-arm2.dts"
-
-&ldb1_phy {
-	status = "okay";
-};
-
-&ldb1 {
-	status = "okay";
-
-	lvds-channel@0 {
-		fsl,data-mapping = "jeida";
-		fsl,data-width = <24>;
-		status = "okay";
-
-		port@1 {
-			reg = <1>;
-
-			lvds0_out: endpoint {
-				remote-endpoint = <&it6263_0_in>;
-			};
-		};
-	};
-};
-
-&i2c0_mipi_lvds0 {
-	#address-cells = <1>;
-	#size-cells = <0>;
-	pinctrl-names = "default";
-	pinctrl-0 = <&pinctrl_i2c0_mipi_lvds0>;
-	clock-frequency = <100000>;
-	status = "okay";
-
-	lvds-to-hdmi-bridge@4c {
-		compatible = "ite,it6263";
-		reg = <0x4c>;
-
-		port {
-			it6263_0_in: endpoint {
-				clock-lanes = <3>;
-				data-lanes = <0 1 2 4>;
-				remote-endpoint = <&lvds0_out>;
-			};
-		};
-	};
-};
-
-&ldb2_phy {
-	status = "okay";
-};
-
-&ldb2 {
-	status = "okay";
-
-	lvds-channel@0 {
-		fsl,data-mapping = "jeida";
-		fsl,data-width = <24>;
-		status = "okay";
-
-		port@1 {
-			reg = <1>;
-
-			lvds1_out: endpoint {
-				remote-endpoint = <&it6263_1_in>;
-			};
-		};
-	};
-};
-
-&i2c0_mipi_lvds1 {
-	#address-cells = <1>;
-	#size-cells = <0>;
-	pinctrl-names = "default";
-	pinctrl-0 = <&pinctrl_i2c0_mipi_lvds1>;
-	clock-frequency = <100000>;
-	status = "okay";
-
-	lvds-to-hdmi-bridge@4c {
-		compatible = "ite,it6263";
-		reg = <0x4c>;
-
-		port {
-			it6263_1_in: endpoint {
-				clock-lanes = <3>;
-				data-lanes = <0 1 2 4>;
-				remote-endpoint = <&lvds1_out>;
-			};
-		};
-	};
-};
-- 
1.7.9.5

